### PR TITLE
Pension 71897 benefit intake polling

### PR DESCRIPTION
**Pull request by wayne-weibel**
## Summary

- add BenefitIntake API polling db entries (FormSubmmission and FormSubmissionAttempt) on success
- Pensions

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/71897

## Testing done

- No form_submission entry

- submit a 527ez (Pension) form locally
- run local vets-api rails console
  - `>  Lighthouse::PensionBenefitIntakeJob.new().perform(ID)` (ID = submitted form id in step above)
  - local database should have an entry in form_submission and form_submission_attempt
- `rspec spec/sidekiq/lighthouse/pension_benefit_intake_job_spec.rb`
  - all tests should pass
 
- same steps as above done as developer testing

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by tblackwe — DISMISSED** [review]

File: Makefile
Comment (rmtolmach) [inline Makefile:1]: @wayne-weibel Can you revert these `Makefile` additions? The Platform is not in a position to maintain these along with the documentation.

File: app/sidekiq/lighthouse/pension_benefit_intake_job.rb [hunk 1]
```diff
@@ -113,11 +113,26 @@ def check_success(response)
...
       else
         raise PensionBenefitIntakeError, response.to_s
       end
     end
 
+    # Insert submission polling entries
+    def form_submission_polling
+      form_submission = FormSubmission.create(
+        form_type: @claim.form_id,
+        form_data: @claim.to_json,
```
Comment (rmtolmach) [inline app/sidekiq/lighthouse/pension_benefit_intake_job.rb:126]: Could this potentially expose veteran's PII?

**Review by rmtolmach — CHANGES_REQUESTED** [review]
Please remove the Makefile changes. Thanks!

File: app/sidekiq/lighthouse/pension_benefit_intake_job.rb (hunk 1 above)
Comment (wayne-weibel) [reply app/sidekiq/lighthouse/pension_benefit_intake_job.rb:126]: Not with the changes here.  The submission or attempt should not be logged (and are not at the moment).

File: Makefile
Comment (wayne-weibel) [reply Makefile:1]: @rmtolmach I would prefer to leave them in for quality-of-life. I can update the documentation for native setup if needed.

Comment (rmtolmach) [reply Makefile:1]: We can't support these now. The Platform Backend folks will discuss whether we want native commands in the Makefile next week. Until then, please remove these. Our Makefile has historically been reserved for Docker setup commands only, so having native commands might confuse things.

File: app/sidekiq/lighthouse/pension_benefit_intake_job.rb (hunk 1 above)
Comment (rmtolmach) [reply app/sidekiq/lighthouse/pension_benefit_intake_job.rb:126]: ok, sounds good!

File: Makefile
Comment (rmtolmach) [reply Makefile:1]: @wayne-weibel see my comment ☝️

**Review by rmtolmach — APPROVED** [review]

File: Makefile
Comment (rmtolmach) [reply Makefile:1]: thanks!

**Pull request merged**